xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/diff_macros.vm

```diff
@@ -142,11 +142,11 @@
     #if (!$documentReference)
       #set ($documentReference = $newDoc.documentReference)
     #end
-    <dt id="#diffAnchor($documentReference)">
+    <dt id="#diffAnchor($documentReference)" #diffDataReference($documentReference)>
       #diffIcon('file-text' 'change')
       $services.localization.render('web.history.changes.summary.documentProperties')
       <span class="hint">
-        ##$escapetool.xml($services.model.serialize($documentReference, 'local'))
+        $escapetool.xml($services.model.serialize($documentReference, 'local'))
```

Comment:
I don't see the value in displaying the document reference here. It looks bad and it creates confusion (we don't display document references in the XWiki UI). When the diff is for a single page then it should be obvious that the changed page properties are of that page. When the diff is for multiple pages (like when we compute the changes for an installed XAR extension) each page gets a heading so it should be easy to know whose page properties are shown in the diff. I think we should simply remove the hint.